src/main/java/thePackmaster/cards/sentinelpack/AbstractSentinelCard.java

```diff
@@ -0,0 +1,14 @@
+package thePackmaster.cards.sentinelpack;
+
+import thePackmaster.cards.AbstractPackmasterCard;
+
+public abstract class AbstractSentinelCard extends AbstractPackmasterCard {
+    public AbstractSentinelCard(String cardID, int cost, CardType type, CardRarity rarity, CardTarget target) {
+        super(cardID, cost, type, rarity, target, CardColor.PURPLE);
```

Comment:
Does this not mean it can spawn for the Watcher?

Reply:
If so, we have that issue elsewhere - the PMClaw card is declared as BLUE.

Reply:
I'll see if they show up on Watcher

Reply:
They do not show up on Watcher, at least not 1 in a sample size of 250 cards.

Reply:
They do indeed appear on Watcher.  I'll make them packmaster color and override the frame.